dnd-troops-service/src/test/java/cz/muni/fi/pa165/w2018/dndtroops/service/facade/HeroFacadeImplTest.java

```diff
@@ -44,9 +41,6 @@
 public class HeroFacadeImplTest extends AbstractTestNGSpringContextTests {
     @Mock
     private HeroService heroService;
-    private GroupService groupService;
```

Comment:
You shouldn't delete these mock, HeroFacade uses all 3 services and need these mock even if right now test doesn't use them directly, we still want mocked services instead of real ones